core/src/ast_builder/expr_list.rs

```diff
@@ -41,10 +60,74 @@ impl<'a> TryFrom<ExprList<'a>> for Vec<Expr> {
                 .iter()
                 .map(translate_expr)
                 .collect::<Result<Vec<_>>>(),
-            ExprList::Exprs(nodes) => nodes
-                .into_iter()
-                .map(TryInto::try_into)
-                .collect::<Result<Vec<_>>>(),
+            ExprList::Exprs(exprs) => {
+                let exprs = exprs.into_owned();
+
+                exprs
+                    .into_iter()
+                    .map(TryInto::try_into)
+                    .collect::<Result<Vec<_>>>()
```

Comment:
<https://github.com/rust-lang/rust-clippy/blob/967f172e256802f9bf0ad5718d9c761da18132a9/clippy_lints/src/methods/unnecessary_to_owned.rs#L198-L239>

It gives an `unnecessary_to_owned` clippy error, and clippy tells you to fix it like this:

```
T.iter().cloned()
```

But it seems rather unnecessary referencing.
The behavior of `into_owned` is only to strip the enum when `Cow::Owned`, whereas `Iterator::cloned` calls `Clone::clone`.

So we separated it like this.